internal/cli/hook_events.go

```
@@ -57,3 +82,13 @@ func printHookEvents(results []core.HookResult) {
 		}
 	}
 }
+
+// withStderrEventStream registers a live-streaming observer on wm that
+// writes each event to stderr, runs fn, then clears the observer. Returns
+// whatever fn returns. stderr is chosen so captured stdout from the hook
+// body is not polluted by event lines.
+func withStderrEventStream(wm *core.WorktreeManager, fn func()) {
```

golangci-lint (2.11.4)

[error] func withStderrEventStream is unused

(unused)